docs/csharp/programming-guide/concepts/linq/introduction-to-pure-functional-transformations.md

```diff
@@ -6,6 +6,12 @@ ms.assetid: 8495c9d9-2d02-4aa0-8a10-9e8794b985fe
 # Introduction to Pure Functional Transformations (C#)
 This section introduces functional transformations, including the underlying concepts and supporting language constructs. It contrasts the object-oriented and functional transformation approaches to programming, including advice on how to transition to the latter. Although functional transformations can be used in many programming scenarios, XML transformation is used here as a concrete example.  
   
+ The [Tutorial: Manipulating Content in a WordprocessingML Document (C#)](../../../../csharp/programming-guide/concepts/linq/shape-of-wordprocessingml-documents.md) tutorial provides a series of examples, each building on the previous one. These examples demonstrate the pure functional transformational approach to manipulating XML.  
+  
+ This tutorial assumes a working knowledge of C#. Detailed semantics of the language constructs are not provided in this tutorial, but links are provided to the language documentation as appropriate.  
```

Comment:
This paragraph and the next one should be combined with the previous one to form a single paragraph.